app/src/main/java/com/example/music_67/data/model/Track.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.music_67.data.model
+
+data class Track(
+    val id: Int,
+    val title: String,
```

Comment:
Với model map dữ liệu từ API, em nên khai báo tất cá trường là Nullable. Riêng với trường id thì có thể để là Non-null
Tương tự với các trường và model khác
```suggestion
    val title: String? = null,
```